aws/table_aws_ssm_association.go

```diff
@@ -72,29 +90,77 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_TIMESTAMP,
 			},
 			{
-				Name:        "overview",
-				Description: "Information about the association.",
-				Type:        proto.ColumnType_JSON,
+				Name:        "last_successful_execution_date",
+				Description: "The last date on which the association was successfully run.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsSSMAssociation,
+			},
+			{
+				Name:        "last_update_association_date",
+				Description: "The date when the association was last updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsSSMAssociation,
 			},
 			{
 				Name:        "schedule_expression",
 				Description: "A cron expression that specifies a schedule when the association runs.",
 				Hydrate:     getAwsSSMAssociation,
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "max_concurrency",
+				Description: "The maximum number of targets allowed to run the association at the same time.",
+				Hydrate:     getAwsSSMAssociation,
+				Type:        proto.ColumnType_STRING,
+			},
```

Comment:
What is the type of the values stored in this column?
Can we convert the value to `int`? As per description it seems that it stores a integer value
`The maximum number of targets allowed`

Reply:
max_concurrency field can be set with percentage also , example 10%, it will be a string.